contracts/src/v0.8/operatorforwarder/dev/AuthorizedReceiver.sol

```diff
@@ -0,0 +1,72 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.19;
+
+import {AuthorizedReceiverInterface} from "../../interfaces/AuthorizedReceiverInterface.sol";
+
+// solhint-disable custom-errors
+abstract contract AuthorizedReceiver is AuthorizedReceiverInterface {
+  mapping(address => bool) private s_authorizedSenders;
+  address[] private s_authorizedSenderList;
+
+  event AuthorizedSendersChanged(address[] senders, address changedBy);
+
+  // @notice Sets the fulfillment permission for a given node. Use `true` to allow, `false` to disallow.
+  // @param senders The addresses of the authorized Chainlink node
+
+  function setAuthorizedSenders(address[] calldata senders) external override validateAuthorizedSenderSetter {
+    require(senders.length > 0, "Must have at least 1 sender");
+    // Set previous authorized senders to false
+    uint256 authorizedSendersLength = s_authorizedSenderList.length;
+    for (uint256 i = 0; i < authorizedSendersLength; i++) {
+      s_authorizedSenders[s_authorizedSenderList[i]] = false;
+    }
+    // Set new to true
+    for (uint256 i = 0; i < senders.length; i++) {
+      require(s_authorizedSenders[senders[i]] == false, "Must not have duplicate senders");
+      s_authorizedSenders[senders[i]] = true;
+    }
+    // Replace list
+    s_authorizedSenderList = senders;
+    emit AuthorizedSendersChanged(senders, msg.sender);
+  }
+
+  // @notice Retrieve a list of authorized senders
+  // @return array of addresses
+
```

Comment:
nit same everywhere, rm empty line

Reply:
these were added by prettier:write

Reply:
That does not match my experience with prettier, we don't have any newlines above functions in any other code